index_trade.html

```diff
@@ -0,0 +1,312 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+  <meta charset='UTF-8'>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <title>Week 4: Replication Project</title>
+  <link rel='stylesheet' href='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css'>
+  <style>
+    body {
+	background-color: #3f3d4c;
+	font-family: 'Trebuchet MS', Helvetica, sans-serif;
+	font-size: 16px;	
+        color: #ff944d;
+        
+    }     
+    a:link, a:visited {
+    color: #ffb1b9;          
+                  
+      }
+    .section {
+      padding: 1em 0;
+    }
+    .header {
+      background: linear-gradient(to bottom, #302e39 0%, #5f5b71 100%); 
+      color: #ff944d;
+      padding: 1em 0 2em;
+    }
+    svg {
+      display: block;
+      margin: 0 auto;
+    }
+    .axis path, .axis line {
+      fill: none;
+      stroke: #ff944d;
+      shape-rendering: crispEdges;
+    }
+    .axis text {
+        font-size: 14px;
+        fill: #ff944d;
+    }
+    .label {
+      font-weight: 700;
+    }
+    .point {
+      stroke: rgb(51, 122, 183);
+      stroke-width: 1;
+      fill: rgba(51, 122, 183, .5);
+      opacity: .5;
+    }
+    .point.filtered {
+      opacity: .05;
+    }
+    .line {
+      stroke: rgb(175, 80, 75);
+      stroke-width: 2;
+    } 
+      
+    .axis .minor line {
+      stroke: #ff944d;
+      stroke-dasharray: 2,2;
+    }      
+      
+    .bar {  
+      fill: steelblue;
+    }     
+       
+    .ybar {
+      fill: #cc0000;
```

Comment:
Another no need to make changes comment, but does it make sense for this histogram to be coded in a different color, as it still corresponds to the same (blue) circles as does the x axis histogram?